.oxlintrc.json

```
@@ -0,0 +1,51 @@
+{
+  "$schema": "https://raw.githubusercontent.com/nicolo-ribaudo/oxc-project.github.io/refs/heads/json-schema/src/public/.oxlintrc.schema.json",
+  "options": {
```

Biome (2.5.0)

[error] The key options was already declared.

(lint/suspicious/noDuplicateObjectKeys)